backend/src/main/java/gov/cdc/usds/simplereport/db/model/Result.java

```diff
@@ -58,8 +58,7 @@ public Result(TestOrder testOrder, SupportedDisease disease, TestResult testResu
   }
 
   /* Copy constructor, used for corrections and removals */
-  public Result(Result originalResult, TestEvent testEvent) {
-    this.testEvent = testEvent;
```

Comment:
Why are we removing this testEvent, it seems unrelated to the other `result` changes?